docs/data-masking-function-list.md

```diff
@@ -0,0 +1,888 @@
+| str | No | The string to be masked | String |
+| mask_char | Yes | The masking character | String |
+
+The `str` accepts an alphanumeric string.
+
+If the `mask_char` is not specified, the default is `X`.
+
+### Returns
+
+A string with the selected characters masked by a specified `mask_char` or that option's default value.
+
+Returns an error if the length of `str` is not correct.
+
+Returns a NULL value if the `str` is in an incorrect length. If the `str` contains a multi-byte character, the `str` is not masked.
+
+### Example
+
+```{.bash data-prompt="mysql>"}
+mysql> SELECT mask_canada_sin('555-555-555');
+```
+
+??? example "Expected output"
+
+    ```{.text .no-copy}
+    +--------------------------------+
+    | mask_canada_sin('555-555-555') |
+    +--------------------------------+
+    | XXX-XXX-XXX                    |
+    +--------------------------------+
+    ```
+
+## mask_iban(str [,mask_char])
+
+
+
+Masks an Internal Bank Account Number (IBAN).
+
+### Parameters
+
+| Parameter | Optional | Description | Type |
+| --- | --- | --- | --- |
+| str | No | The string to be masked | String |
+| mask_char | Yes | Character used for masking | String |
+
+The `str` accepts either of the following:
+
+* No separator symbol
+* Groups of four characters. These groups can be separated by a space or any separator character.
+
+A `str` 
+The default value for `mask_char` is `*`.
+
+### Returns
+
+Returns the masked string.
+
+Returns an error if `str` length is incorrect.
+
+If the `str` contains a multi-byte character, the `str` is not masked.
+
+### Example
+
+```{.bash data-prompt="mysql>"}
+mysql> SELECT mask_iban('DE27 1002 02003 77495 4156');
+```
+
+??? example "Expected output"
+
+    ```{.text .no-copy}
+    +---------------------------------------------+
+    | mask_iban('DE27 1002 02003 77495 4156')     |
+    +---------------------------------------------+
+    |DE** **** **** **** ****                     |
+    +---------------------------------------------+
+    ```
+
+## mask_inner(str, margin1, margin2 [,mask_char])
+
+
+
+Returns the string where a selected inner portion is masked with a substitute character.
+
+### Parameters
+
+| Parameter | Optional | Description | Type |
+| --- | --- | --- | --- |
+| string | No | The string to be masked | String |
+| margin1 | No | The number of characters on the left end of the string to remain unmasked | Integer |
+| margin2 | No | The number of characters on the right end of the string to remain unmasked | Integer |
+| mask_char | Yes | The masking character | String |
+
+If the `margin1` cannot be a negative number. If the marin1 value is 0, then all of the left end characters are unmasked.
+
+If the `margin2` cannot be a negative number. If the marin1 value is 0, then all of the right end characters are unmasked.
+
+If the sum of `margin1` and `margin2` is greater than the string length, no masking occurs.
+
+If the `mask_char` is not specified, the default is 'X'.
+
```

Comment:
mask_char can be a multibyte character in any character set (not necessarily identical to the character set of str)